lib/sprinkler_web/tortoise/mqtt_handler.ex

```diff
@@ -0,0 +1,42 @@
+defmodule SprinklerWeb.Tortoise.MqttHandler do
+  use Tortoise.Handler
+
+  def init(args) do
+    {:ok, args}
+  end
+
+  def connection(_status, state) do
+    # `status` will be either `:up` or `:down`; you can use this to
+    # inform the rest of your system if the connection is currently
+    # open or closed; tortoise should be busy reconnecting if you get
+    # a `:down`
+    {:ok, state}
+  end
+
+  def handle_message(["rs", client_id, "sensors"], payload, state) do
```

Comment:
The terminology for IoT is telemetry
```suggestion
  def handle_message(["rs", client_id, "telemetry"], payload, state) do
```